lexactivator/src/main/java/com/cryptlex/android/lexactivator/LexActivatorException.java

```diff
@@ -514,6 +534,28 @@ public static String getErrorMessage(int errorCode) {
      */
     public static final int LA_E_IP = 82;
 
+    /*
+     * CODE: LA_E_RELEASE_VERSION
+     *
+     * MESSAGE: Invalid release version. Make sure the release version uses
+     * the following formats: x.x, x.x.x, x.x.x.x (where x is a number).
+     */
+    public static final int LA_E_RELEASE_VERSION = 84;
```

Comment:
@adnan-kamili 
The reason for skipping code 83 in android is because 83 is reserved for LA_E_CONTAINER in others. Is the 83 deliberately not mentioned or have we missed it earlier?